src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/PanelStatus.tsx

```diff
@@ -3,6 +3,17 @@ import { getIsIndexed } from "ui/reducers/app";
 import { AnalysisError, AnalysisPayload } from "ui/state/app";
 import { getExecutionPoint } from "../../../selectors";
 
+const getPreviousPointsCount = (analysisPoints: AnalysisPayload, executionPoint: string | null) => {
+  if (!executionPoint) {
+    return "…";
```

Comment:
This branch feels off to me.

I guess fundamentally, I think that a function named "get count" should always return a number.

In this case, we're trying to differentiate between "it's this specific number in the set of numbers" and "it's not in the set of numbers" then maybe `-1` (like `indexOf`) would be a good value to signal that?

Reply:
Oh good call, yeah let me clean that up. And then interpret the `-1` as "…"